bower.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "polymerfire",
-  "version": "0.9.4",
+  "name": "polymerfire-extended",
+  "version": "0.9.5",
   "authors": [
```

Comment:
Hey, you can use Bower's `resolutions` for that, enabling you to download your version `fernandezpaco/polymerfire` whenever it requires `polymerfire`.

If you keep your pull request clean it can be merged, otherwise it needs to be replicated, because this commit cannot be merged this whole pull request cannot be merged.